tests/EdFi.Ods.WebApi.CompositeSpecFlowTests/Steps.cs

```diff
@@ -12,6 +12,7 @@
 using System.Threading.Tasks;
 using ApprovalTests;
 using ApprovalTests.Reporters;
+using Castle.Core.Internal;
```

Comment:
Should not be using extension methods from `Castle.Core.Internal`.

Reply:
removed